angular-client/angular.json

```diff
@@ -20,15 +20,15 @@
             "tsConfig": "tsconfig.app.json",
             "assets": ["src/favicon.ico", "src/assets"],
             "styles": ["@angular/material/prebuilt-themes/deeppurple-amber.css", "src/styles.css"],
-            "scripts": []
+            "scripts": ["node_modules/apexcharts/dist/apexcharts.min.js"]
```

Comment:
you're pulling a script file from the node_modules, is this just how they have u set it up?

Reply:
yeah according to the apex charts documentation